### DISCUSSION TITLE
liquibase errors when starting nzyme

### DISCUSSION THREAD

**Discussion by MoerkyMark** [nzymedefense/nzyme#536] [Questions]
First time trying out nzyme and I followed the very good installation documentation step by step. But when starting nzyme I get "Cannot load service: liquibase.parser.ChangeLogParser" errors. Following also Cannot load service: liquibase.hub.HubService. I'm using a Raspberry PI4 with debian "buster".

**Comment by lennartkoopmann** [comment]
* Did you install nzyme via the official DEB packages?
* What Java version are you running?
* Please post the full nzyme log
